spec/features/pictures_feature_spec.rb

```diff
@@ -0,0 +1,80 @@
+require 'rails_helper'
+
+feature 'Pictures' do
+  let(:user2) { create :user, :second }
+  let!(:user){User.create(name: 'Betty Draper', email: 'betty@drapers.com',
+                          username: 'bettydraper', password: 'janedoee',
+                          password_confirmation: 'janedoee')}
+  let(:picture) {Picture.create(description: 'My first picture!',
```

Comment:
Style/BlockDelimiters: Avoid using `{...}` for multi-line blocks. (https://github.com/bbatsov/ruby-style-guide#single-line-blocks)<br>Style/MultilineBlockLayout: Block body expression is on the same line as the block start.